### PR TITLE
Select kernel in test and ignore output count

### PR DESCRIPTION
**Pull request by DonJayamanne**

### CODE REVIEW

File: src/kernels/kernelFinder.ts [hunk 1]
```diff
@@ -80,49 +77,45 @@ export class KernelFinder implements IKernelFinder {
         // List kernels might be called after finders or connections are removed so just clear out and regenerate
         this.connectionFinderMapping.clear();
         let finders: IContributedKernelFinder<KernelConnectionMetadata>[] = [];
-        if (this.featureManager.features.kernelPickerType === 'Insiders') {
```
Comment (DonJayamanne) [inline src/kernels/kernelFinder.ts:83]: Remove if condition and apply fixes to the old kernel picker as well

File: src/test/datascience/debugger.vscode.test.ts [hunk 2]
```diff
@@ -69,13 +69,19 @@ suite('Run By Line @debugger', function () {
         sinon.restore();
 
         if (!isWeb() && !IS_REMOTE_NATIVE_TEST()) {
-            await api.serviceContainer
+            const controller = await api.serviceContainer
                 .get<IControllerDefaultService>(IControllerDefaultService)
-                .computeDefaultController(undefined, 'jupyter-notebook');
+                .computeDefaultController(undefined, 'jupyter-notebook'); // Create an editor to use for our tests
+            await createEmptyPythonNotebook(disposables, undefined, false);
+            await commands.executeCommand('notebook.selectKernel', {
```
Comment (DonJayamanne) [inline src/test/datascience/debugger.vscode.test.ts:76]: manually test kernel for tests

File: src/test/datascience/notebook/executionService.vscode.test.ts [hunk 3]
```diff
@@ -129,24 +129,10 @@ suite('Kernel Execution @kernelCore', function () {
...
 
         assert.isAtLeast(cell.executionSummary?.executionOrder || 0, 1);
-        assert.strictEqual(
-            cell.outputs.length,
-            1,
-            `Cell should have one output, but has ${cell.outputs.length}, ${cell.outputs
-                .map((item) => `Cell Output items => ${item.items.map((oi) => oi.mime).join('')}`)
-                .join(', ')}`
-        );
-        assert.strictEqual(
```
Comment (DonJayamanne) [inline src/test/datascience/notebook/executionService.vscode.test.ts:139]: no point testing number of outputs as the code to merge outputs is in vscode and these tests don't rely on vscode controllers

File: src/notebooks/controllers/controllerLoader.ts [hunk 4]
```diff
@@ -85,6 +85,12 @@ export class ControllerLoader implements IControllerLoader, IExtensionSyncActiva
                     this.interpreters,
                     this.registration
                 ).catch(noop);
+                createActiveInterpreterController(
```
Comment (DonJayamanne) [inline src/notebooks/controllers/controllerLoader.ts:88]: Always create the active interpreter controller for both NB and IW

File: src/notebooks/controllers/controllerLoader.ts [hunk 5]
```diff
@@ -105,6 +111,12 @@ export class ControllerLoader implements IControllerLoader, IExtensionSyncActiva
                         this.interpreters,
                         this.registration
                     );
+                    await createActiveInterpreterController(
```
Comment (DonJayamanne) [inline src/notebooks/controllers/controllerLoader.ts:114]: Always create the active interpreter controller for both NB and IW

**Review by sandy081 — APPROVED** [review]

**Pull request merged**